src/Workspaces/Core/Portable/Workspace/Solution/SolutionState.cs

```diff
@@ -823,19 +834,6 @@ public SolutionState WithHasAllInformation(ProjectId projectId, bool hasAllInfor
             return this.ForkProject(newProject);
         }
 
-        private static async Task<Compilation> ReplaceSyntaxTreesWithTreesFromNewProjectStateAsync(Compilation compilation, ProjectState projectState, CancellationToken cancellationToken)
-        {
-            var syntaxTrees = new List<SyntaxTree>(capacity: projectState.DocumentIds.Count);
-
-            foreach (var documentState in projectState.OrderedDocumentStates)
-            {
-                cancellationToken.ThrowIfCancellationRequested();
-                syntaxTrees.Add(await documentState.GetSyntaxTreeAsync(cancellationToken).ConfigureAwait(false));
-            }
-
-            return compilation.RemoveAllSyntaxTrees().AddSyntaxTrees(syntaxTrees);
-        }
```

Comment:
never called?

Reply:
Correct.

Reply:
It was only called by the translation action. I just inlined it.